src/formatDate.js

```diff
@@ -8,7 +8,59 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDate = date.split(fromFormat[3]);
```

Comment:
It's better to do it through fromFormat.length - 1

Reply:
Okay!

Reply:
In this very case, this is not critical, as what is exactly the aim of using `fromFormat.length - 1`?
Personally, I would use access by explicit index